modules/apps/portal-search/portal-search-web/src/main/resources/com/liferay/portal/search/web/internal/category/facet/template/dependencies/portlet_display_template_compact.ftl

```diff
@@ -0,0 +1,65 @@
+<@liferay_ui["panel-container"]
+	extended=true
+	id="${namespace + 'facetAssetCategoriesPanelContainer'}"
+	markupView="lexicon"
+	persistState=true
+>
+	<@liferay_ui.panel
+		collapsible=true
+		cssClass="search-facet"
+		id="${namespace + 'facetAssetCategoriesPanel'}"
+		markupView="lexicon"
+		persistState=true
+		title="category"
+	>
+		<@liferay_aui.form
+			method="post"
+			name="categoryFacetForm"
+		>
+			<@liferay_aui.input
+				autocomplete="off"
+				name=htmlUtil.escape(assetCategoriesSearchFacetDisplayContext.getParameterName())
+				type="hidden"
+				value=assetCategoriesSearchFacetDisplayContext.getParameterValue()
+			/>
+			<@liferay_aui.input
+				cssClass="facet-parameter-name"
+				name="facet-parameter-name"
+				type="hidden"
+				value=assetCategoriesSearchFacetDisplayContext.getParameterName()
+			/>
+
```

Comment:
Element blocks on the same level should have a new line between them. (Unless the elements are similar like hidden inputs and on 1 line each.)